control/init.sh

```diff
@@ -17,10 +17,20 @@ function realpath() {
 
 
 if [ -z ${CI_TEST} ]; then
-    if [ ! -f $VENV/bin/activate ]; then
-        if [[ $CCHQ_VIRTUALENV == *3.10* ]]; then
-          # use venv because 3.10 setup includes installing python3.10-venv
-          python3.10 -m venv $VENV
+    if [[ ! -f $VENV/bin/activate ]]; then
```

Comment:
Will this always use the existing (3.6) `cchq` virtualenv if it exists?

Reply:
Oh good catch. I think the `if [[ ! -f $VENV/bin/activate ]]; then` check is completely unnecessary here as the script hasn't even decided which venv to use yet. Will update that!

Reply:
See 42925c49685db2325c02596168cb1649b50f9c78